### PR TITLE
Made a javascript to typescript refactor

### PR DESCRIPTION
**Pull request by Burnedduk3**
Made a refactor from JS to TS and tried to fix some styling issues.

### CODE REVIEW

File: packages/css/webpack.config.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 var path = require('path');
 
 const OUTPUT_PATH = path.resolve(__dirname, '.');
-const ENTRY_POINT = path.resolve(__dirname, 'src/index.js');
+const ENTRY_POINT = path.resolve(__dirname, 'src/index.ts');
```
Comment (Jkierem) [inline packages/css/webpack.config.js]: This file does not exist. CSS package uses js. If you want to migrate it to TS you will need to add typings to `src/atoms/select/index.js`, `src/index.js`, `src/atoms/datepicker/index.js`, `src/atoms/icons/index.js`, `src/utility/index.js` and `src/atoms/index.js` and add typescript to the webpack config. I would advise against this. The behaviors.js is not meant to be used as a library, rather as a side effect (that has no visible effects on global types and has no exports) so other than dev experience, there is no benefit to consumers of it.

Comment (Burnedduk3) [reply packages/css/webpack.config.js]: I would take a look into this, this appears to be a refactor made by webstorm.

File: packages/sandbox/package.json [hunk 2]
```diff
@@ -8,14 +8,15 @@
     "@testing-library/user-event": "^12.6.3",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
-    "react-scripts": "^4.0.2"
+    "react-scripts": "^4.0.2",
+    "typescript": "^4.6.2"
```
Comment (suabochica) [inline packages/sandbox/package.json]: please define `typescript` as a `devDependency`. This [post](https://dev.to/philippurwoko/should-i-install-typescript-globally-or-at-devdepencendy-468o) can help us to understand the reason

Comment (Burnedduk3) [reply packages/sandbox/package.json]: sure :D

File: packages/sandbox/src/App.tsx [hunk 3]
```diff
@@ -1,180 +1,268 @@
-import React from 'react';
+import React, {Component, FC} from 'react';
```
Comment (suabochica) [inline packages/sandbox/src/App.tsx:1]: first of all, We really appreciate this update, thanks for dedicated your time. The PR looks good to me but a I will ask you a last favor 🙊 . Can you please order the components alphabetically 🙏🏾  ? Is to keep the same convention of the folder `packages/css/src/atoms`

**Review by suabochica — APPROVED** [review]

File: packages/sandbox/src/App.tsx (hunk 3 above)
Comment (Burnedduk3) [reply packages/sandbox/src/App.tsx:1]: done :D

Comment (suabochica) [reply packages/sandbox/src/App.tsx:1]: Thanks!

**Pull request merged**